manifests/w/WonderIdea/WanCaiVR/1.5.1/WonderIdea.WanCaiVR.installer.yaml

```
@@ -0,0 +1,22 @@
+# Created with YamlCreate.ps1 Dumplings Mod
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: WonderIdea.WanCaiVR
+PackageVersion: 1.5.1
+InstallerType: inno
+Scope: machine
+UpgradeBehavior: install
+FileExtensions:
+- ac
+- hand
+ProductCode: WanCaiVR_is1
+ReleaseDate: 2025-02-17
+Installers:
+- Architecture: x86
+  InstallerUrl: https://download.wancaiinfo.com/wancaivr/static_file/wancaivr_v1.5.1_x86_setup-cn.exe
+  InstallerSha256: A6F27F7000D84BB3F6CCD744B4FA69976CD12121AABED1533AD64BAF40BEA0E8

```

```suggestion
 InstallerSha256: 95971A4275A9AA69A0C50F0F5662DF0FD0D7835E18FDF9B59500E536F15F9F67
```

(Automated response - build 982.)